server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -0,0 +1,182 @@
+Feature: GetMessages method
+  As a James user
+  I want to be able to retrieve my messages
+
+  Background:
+    Given a domain named "domain.tld"
+    And a current user with username "username@domain.tld" and password "secret"
+    And the current user has a mailbox named "inbox"
+
+  Scenario: Retrieving messages with a non null accountId should return a NotSupported error
+    When the user is getting his messages and gives a non null accountId
+    Then an error "Not yet implemented" is returned
+
+  Scenario: Unknown arguments should be ignored when retrieving messages
+    When the user is getting his messages and gives unknown arguments
+    Then no error is returned
+    And the list of unknown messages is empty
+    And the list of messages is empty
+
+  Scenario: Retrieving messages with invalid argument should return an InvalidArguments error
+    When the user is getting his messages and gives invalid argument
+    Then an error "invalidArguments" is returned
+    And the description is "N/A (through reference chain: org.apache.james.jmap.model.Builder["ids"])"
```

Comment:
but we should focus on our gherkin quality : there's no way this description could be correct, if you think we don't have time to fix it, fix the test anyway and mark it as ignored